specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json

```diff
@@ -1769,6 +1769,10 @@
             "name": "WindowsPatchAssessmentMode",
             "modelAsString": true
           }
+        },
+        "automaticByPlatformSettings": {
+          "$ref": "#/definitions/WindowsAutomaticByPlatformPatchSettings",
```

Comment:
Based on the pattern set in CRP (and here on line 1753 'WindowVMGuestPatchMode'), the desired definition name would be 'WindowsVMGuestPatchAutomaticByPlatformSettings'

And similarly for Linux.

This was what the class name would have been internally (see in original PR in CRP 5591183, iteration 1) if not for a one-off convergence for this based on a design discussion with Najam.

Reply:
Resolved